packages/angular/projects/ui-angular/src/lib/primitives-poc/view/view-component/ComponentSetting.ts

```diff
@@ -0,0 +1,16 @@
+export class ComponentSettings {
+  /** Component selector String */
+  public static COMPONENTNAME = '[amplify-view]';
+
+  public static PROPMAP = {
+    arialabel: 'aria-label',
+    ariavaluetext: 'aria-valuetext',
+    backgroundcolor: 'background-color',
+    borderradius: 'border-radius',
+  };
```

Comment:
Looks like this isn't actually used in view component. Is this needed?

Reply:
Removed.